acceptance/tests/ensure_puppet-agent_paths.rb

```diff
@@ -118,6 +118,19 @@ def config_options(agent)
   end
 end
 
+step 'test puppet hiera file install' do
+  agents.each do |agent|
+    hiera_config = config_options(agent).select {|v| v[:name] == :hiera_config}.first[:expected]
+    env_path = config_options(agent).select {|v| v[:name] == :environmentpath}.first
+    hiera_env_config = "#{env_path[:expected]}/production/hiera.yaml"
+
+    [hiera_config, hiera_env_config].each do |file|
+      on(agent, "cat '#{file}'") do |cat_result|
+        assert_match(/version: 5/, cat_result.stdout, "Expected hiera version to be set in #{file}")
```

Comment:
We may want to future proof the version number match to prevent this test from failing when we bump it. Possibly match `/version [5-9]/`.